swap_meet/vendor.py

```diff
@@ -1,2 +1,66 @@
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        if inventory is None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
```

Comment:
Nice handling of the default empty inventory and remembering that we shouldn't pass a mutable object as a default param. 

Notice that when we have the pattern

```python
if some_condition:
    my_var = "some value"
else:
    my_var = "other value"
```

This can be re-written as a ternary, as:

`my_var = "some value" if some_condition else "other value"`

In this situation, that would look like

```python
self.inventory = [] if inventory is None else inventory
```